pkgdown/_pkgdown.yml

```diff
@@ -155,13 +155,13 @@ reference:
 navbar:
   structure:
     left:
+    right:
```

Comment:
this I'm not sure the maintainer wants to change. that's something pkgdown website maintainers can change as they will. Unless it looks really awful with the current design?

Reply:
I wonder if we can achieve the designer's intention by treating all "left" items as if they were "right" items. I'd rather not adapt `_pkgdown.yml` in every package that uses the template.

Reply:
Well in that case we'd need to provide a wrapper around `pkgdown::build_site()` within cynkratemplate. That's what rotemplate, the rOpenSci's pkgdown template, does. But it means we'd have to tweak all package repos to use that... so in the end, easier to tweak YAML configs.

Reply:
Please:

- No wrapper
- No config changes

Pretty sure we can adapt the template, let's discuss there.

Reply:
Ok, let me tweak the cynkratemplate PR